packages/app/components/Queue/QueueCard.tsx

```diff
@@ -31,47 +54,121 @@ const CenterRow = styled(Row)`
 `;
 
 interface QueueCardProps {
+  role: Role;
+  helping: boolean;
   rank: number;
   name: string;
   questionType: QuestionType;
   waitTime: number;
   status: string;
+  onOpen: (name: string) => void;
 }
 
 export default function QueueCard({
+  role,
+  helping,
   rank,
   name,
   questionType,
   waitTime,
   status,
+  onOpen,
 }: QueueCardProps) {
   return (
-    <HorizontalCard>
-      <CenterRow justify="space-between">
-        <Col span={1}>
-          <Text>{rank}</Text>
-        </Col>
-        <Col xs={16} sm={11} lg={6}>
-          <CenterRow>
-            <Avatar icon={<UserOutlined />} />
-            <NameContainer>
-              <Text>{name}</Text>
-            </NameContainer>
+    <div>
```

Comment:
is it possible to either:
- split this up into three separate things so they're more readable
- localize the changes? I know there are quite a few values in here that're tricky to convey (such as some cards having xxl widths and some not), but I feel like that'd be nicer to read than this, but idk if that's just me

Reply:
yeah for sure! i think that'd make things more readable. i'm not sure exactly what you're suggesting, do you think separating each case into three different render methods would be sufficient?

Reply:
i just pushed my changes that address this but i could've misunderstood your request, just reopen this issue if more can be done!